src/Test.UnitTests.Sarif.WorkItems/SarifWorkItemFilerTests.cs

```diff
@@ -105,6 +106,41 @@ public void WorkItemFiler_ValidatesSarifLogFileLocationArgument()
             action.Should().Throw<ArgumentNullException>();
         }
 
+        [Fact]
+        public void WorkItemFiler_SarifLogResultsShouldNotBeFiled()
+        {
+            SarifLog sarifLog = TestData.CreateSimpleLog();
+
+            // Set all results baseline state to 'Unchanged' so no work item should be filed.
+            sarifLog.Runs.SelectMany(run => run.Results).ForEach(result => result.BaselineState = BaselineState.Unchanged);
+
+            SarifWorkItemContext context = CreateAzureDevOpsTestContext();
+
+            context.SplittingStrategy = SplittingStrategy.None;
+
+            int numberOfResults = 0;
+            context.SetProperty(ExpectedWorkItemsCount, numberOfResults);
+            context.SetProperty(ExpectedFilingResult, FilingResult.None);
+
+            TestWorkItemFiler(sarifLog, context, true);
+        }
+
+        [Fact]
+        public void WorkItemFiler_SarifLogResultsAreEmpty()
+        {
+            SarifLog sarifLog = TestData.CreateEmptyRun();
+
+            SarifWorkItemContext context = CreateAzureDevOpsTestContext();
+
+            context.SplittingStrategy = SplittingStrategy.None;
+
+            int numberOfResults = 0;
+            context.SetProperty(ExpectedWorkItemsCount, numberOfResults);
+            context.SetProperty(ExpectedFilingResult, FilingResult.None);
+
+            TestWorkItemFiler(sarifLog, context, true);
```

Comment:
>true [](http://example.com/codeflow?start=49&length=4)

As a rule don't embed literals in method calls. What does true here mean? We don't know.

Instead, qualify the literal with the parameter name. If the parameter name is good, the code will be cleaner:

TestWorkItemFiler(sarifLog, context, adoClient: true)